src/test/ui/consts/const-eval/const_panic.stderr

```diff
@@ -68,7 +68,7 @@ error[E0080]: evaluation of constant value failed
 LL | const Y_CORE: () = core::unreachable!();
    |                    ^^^^^^^^^^^^^^^^^^^^ the evaluated program panicked at 'internal error: entered unreachable code', $DIR/const_panic.rs:30:20
    |
-   = note: this error originates in the macro `core::unreachable` (in Nightly builds, run with -Z macro-backtrace for more info)
+   = note: this error originates in the macro `$crate::panic::unreachable_2015` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
Previously it was `core::unreachable` and now `$crate::panic::unreachable_2015`, reference to `$crate` as rust library in error message doesn't look good.

Reply:
That's not really a problem of this PR. There are already many cases of this for other macros. Here's two examples: https://github.com/rust-lang/rust/blob/1e4067957bd5d0e12c1657e720903209ecc291dc/src/test/ui/consts/const-eval/const_panic.stderr#L7 https://github.com/rust-lang/rust/blob/1e4067957bd5d0e12c1657e720903209ecc291dc/src/test/ui/consts/const-eval/issue-43197.stderr#L42

Might be nice to improve this, but that's not something to block this PR on.

Reply:
Oh, didn't notice, but yes, maybe this should be improved.